fairworkflows/fairstep.py

```diff
@@ -217,34 +225,34 @@ def is_script_task(self, value: bool):
 
     def _get_variable(self, var_ref: Union[rdflib.term.BNode, rdflib.URIRef]) -> FairVariable:
         """Retrieve a specific FairVariable from the RDF triples."""
-        var_type_objs = self._rdf.objects(var_ref, RDF.type)
-        var_types = [var_type for var_type in var_type_objs
-                     if isinstance(var_type, rdflib.term.Literal)]
-        if len(var_types) > 0:
-            var_type = str(var_types[0])
-        else:
-            var_type = None
+        rdfs_comment_objs = list(self._rdf.objects(var_ref, RDFS.comment))
+        computational_type = str(rdfs_comment_objs[0])
+
+        sem_type_objs = self._rdf.objects(var_ref, RDF.type)
```

Comment:
The var_ref also has a RDF.type PPLAN.Variable. We need to exclude this from here as it is not the semantic type of the variable. (and maybe add a test for this since we now missed it).

Reply:
Done.